index.html

```diff
@@ -342,6 +342,16 @@ <h2>
           `{name: "push", userVisibleOnly: false}` is [=PermissionDescriptor/stronger than=]
           `{name: "push", userVisibleOnly: true}`.
         </p>
+        <p>
+          When the permission state changes (for example, it is revoked or re-granted), the <a>user
```

Comment:
This should probably use the revocation hook: https://w3c.github.io/permissions/#dfn-permission-revocation-algorithm

Would be nice if we can decide whether this should be on revocation, regranted, or both.

Reply:
I added a note below, clarifying why it can make sense to leave this up to the implementor.

Reply:
(This comment is not about the note though)

Reply:
And maybe also worth getting feedback from Permisssion API people about whether it's a good idea to fire at regrant at all. I think probably there's a reason why there's no event for permission change?

cc @trikolon

Reply:
(I'm told that @bakulf would know better about permission API, so redirecting the question to him.)